ipython_widgets/static/widgets/js/widget.js

```diff
@@ -810,7 +814,7 @@ define(["nbextensions/widgets/widgets/js/manager",
     };
 
     // For backwards compatability.
-    $.extend(IPython, widget);
```

Comment:
I **think** so.  That too has been deprecated since 3.0, but unlink the others, doesn't have a deprecation warning.  Once we started using requirejs is when that got placed there.